packages/create-bison-app/template/jest.config.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
       },
     },
   },
+  transformIgnorePatterns: ['/node_modules/(?!(@swc|@trpc))/'],
```

Comment:
This is necessary for tRPC to work properly. Normally, Jest doesn't transform node_modules, but one of the tRPC packages has some syntax that Jest doesn't understand natively. This makes it so only the tRPC stuff is transformed by Jest.